File: src/main/java/com/github/gossie/paddysbot/PaddysBotApp.java
package com.github.gossie.paddysbot;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.web.servlet.ServletComponentScan;

@SpringBootApplication
@ServletComponentScan
public class PaddysBotApp {

    public static void main(String[] args) {
        SpringApplication.run(PaddysBotApp.class, args);
    }

}

File: src/main/kotlin/com/github/gossie/paddysbot/PaddysBotApp.kt
package com.github.gossie.paddysbot

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.boot.web.servlet.ServletComponentScan

@SpringBootApplication
@ServletComponentScan
class PaddysBotApp {

}

fun main(args: Array<String>) {
    runApplication<PaddysBotApp>(*args)
}
